src/methods/lombscargle.jl

```diff
@@ -0,0 +1,89 @@
+using LombScargle
+
+export LS
+"""
+LS(t; tol=1, N_total=10000, N_acc=2000,q=1)
+Compute a surrogate of an irregular time series with supporting time steps `t` based on the simulated annealing algorithm described in [^SchreiberSchmitz1999].
```

Comment:
Be explicit that it is the time axis that is irregular. As stated now, it is not clear whether the data values or their (time) indices are irregular.